Packet++/src/IPv6Layer.cpp

```diff
@@ -187,7 +187,7 @@ void IPv6Layer::removeAllExtensions()
 
 bool IPv6Layer::isFragment() const
 {
-	IPv6FragmentationHeader* fragHdr = getExtensionOfType<IPv6FragmentationHeader>();
+	IPv6FragmentationHeader const* fragHdr = getExtensionOfType<IPv6FragmentationHeader>();
```

Comment:
why not `return getExtensionOfType<IPv6FragmentationHeader>() != nullptr`

Reply:
Good idea. Added 94f6a17